### PR TITLE
attempt for fix of #2153

### PR DESCRIPTION
**Pull request by Karsten12**

### CODE REVIEW

**Comment by ddebrunner** [comment]
Please try to have commit messages reflect what was actually done, anyone looking later might not have an immediate access to what 2153 means.

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py [hunk 1]
```diff
@@ -161,22 +162,24 @@ def _lsappconfig_parser(subparsers):
...
+    locale.setlocale(locale.LC_ALL, '')
+    LOCAL_TIMEZONE = datetime.datetime.now(datetime.timezone.utc).astimezone().tzinfo
 
     # If default output format
     if cmd_args.fmt == '%Tf':
         print('{: <20} {:<20} {:<30} {:<30} {:<20}'.format("Id", "Owner", "Created", "Modified", "Description"))
         for config in configs:
-            createDate = datetime.datetime.fromtimestamp(config.creationTime/1000).strftime("%m/%d/%Y, %I:%M %p ") + "GMT"
-            lastModifiedDate = datetime.datetime.fromtimestamp(config.lastModifiedTime/1000).strftime("%m/%d/%Y, %I:%M %p ") + "GMT"
+            createDate = datetime.datetime.fromtimestamp(config.creationTime/1000).replace(tzinfo=LOCAL_TIMEZONE).strftime("%x %X %Z")
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py]: `datetime.datetime.fromtimestamp` already returns a local time so the `replace` is not needed.

https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py [hunk 2]
```diff
@@ -161,22 +162,24 @@ def _lsappconfig_parser(subparsers):
 def _lsappconfig(instance, cmd_args):
     """view appconfigs"""
     configs = instance.get_application_configurations()
+    locale.setlocale(locale.LC_ALL, '')
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py]: Don't think these two lines are needed, see other comment.

Comment (Karsten12) [reply com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py]: Resolved: Needed

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py (hunk 1 above)
Comment (Karsten12) [reply com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py]: Resolved: datetime.datetime.fromtimestamp not aware of timezone, need LOCAL_TIMEZONE

**Pull request merged**